### DISCUSSION TITLE
deployed app not working

### DISCUSSION THREAD

**Discussion by itpemerintahan** [Budibase/budibase#769] [Help]
new to budibase , try do deploy with some default content ( from budibase editor ) 
nothing working 
https://app_68344de9a7db422aa9fdb50ee4acb19d.app.budi.live/app_68344de9a7db422aa9fdb50ee4acb19d

![image](https://user-images.githubusercontent.com/73216271/96682423-fde1de80-13a2-11eb-8824-ee252c533b7d.png)

**Comment by shogunpurple** [comment]
Hi @itpemerintahan - thanks for taking the time to raise an issue. We are aware of some issues with the deployment functionality. We will be pushing some updates today that should help. You will get a notification in your builder to update to the latest version when that happens, then you should be able to deploy your app successfully.

**Comment by AnthonyTwoBears** [comment]
Hi BudiBase,

Just having a play with the app and deployment. How/Where would you like bugs/issues to be reported?

I'm on a mac, created a couple of tables and a relationship. getting some odd behaviours and 503 Errors using the site...

T.

**Reply by joebudi** [reply]
Hey @AnthonyTwoBears 

Thanks for reaching out. We'd appreciate it if you reported bugs/issues [here](https://github.com/Budibase/budibase/issues)

Screenshots and Loom videos are super helpful when it comes to bugs/issues and help the team debug easier.

Thanks,

Joe
